invenio_app_rdm/config.py

```diff
@@ -263,12 +263,15 @@ def files_rest_permission_factory(obj, action):
 sending the response to the client. Set to False, in case of doubt.
 """
 
-# Invenio-Security-Invenio
+# Flask-Security
 # ========================
 
 SECURITY_EMAIL_SUBJECT_REGISTER = _("Welcome to Invenio App RDM!")
 """Email subject for account registration emails."""
 
+SECURITY_POST_LOGOUT_VIEW = "/oauth/logout"
```

Comment:
I would avoid putting this here by default, given that maybe no oauth is configured (only local login). It probably does not hurt, but maybe better to add it to the doc [here](https://inveniordm.docs.cern.ch/customize/authentication/)?

Reply:
with only local accounts I think this would break, to be tested?

Reply:
no it should not, because if you have no remotes configured, it will not redirect (if I am not mistaken). However, I will remove it, and document it instead.

Reply:
Yes, it wouldn't break, but I will document it instead :+1: